staging/src/k8s.io/kube-scheduler/config/v1/types.go

```diff
@@ -185,9 +185,9 @@ type ExtenderTLSConfig struct {
 	CAData []byte `json:"caData,omitempty"`
 }
 
-// Extender holds the parameters used to communicate with the extender. If a verb is unspecified/empty,
+// LegacyExtender holds the parameters used to communicate with the extender. If a verb is unspecified/empty,
 // it is assumed that the extender chose not to provide that extension.
-type Extender struct {
+type LegacyExtender struct {
```

Comment:
why was this type renamed? doing so disrupts the autoconversion generation

Reply:
Yes, it was done to have a clearer name and avoid confusion. There is no intention to ever change this `LegacyExtender` as it's going to be removed in the future with the policy config after Scheduler component config graduates to GA.

Reply:
+1 Marking the type as `LegacyExtender` should be a signal for devs that we don't intent to change this type anymore.